test/WebSites/RazorPagesClassLibrary/xunit.runner.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
You shouldn't need this. This is only required on the functional test project. I'll take care of it if you want.